### PR TITLE
fix(server): Fix replication logs

### LINKED ISSUE #529: Replication TODOs
1. Improve replica error handling and cancellation
2. Allow running replication on single threaded instances
3. Update various information for the info command

### PR DESCRIPTION
**Pull request by dranikpg**
Fixes #529

### CODE REVIEW

File: src/server/rdb_load.cc [hunk 1]
```diff
@@ -2024,31 +2024,31 @@ error_code RdbLoader::HandleAux() {
...
-      LOG(INFO) << "RDB has an AOF tail";
+      DLOG(INFO) << "RDB has an AOF tail";
     }
   } else if (auxkey == "redis-bits") {
     /* Just ignored. */
   } else {
     /* We ignore fields we don't understand, as by AUX field
      * contract. */
-    LOG(WARNING) << "Unrecognized RDB AUX field: '" << auxkey << "'";
+    DLOG(WARNING) << "Unrecognized RDB AUX field: '" << auxkey << "'";
```
Comment (romange) [inline src/server/rdb_load.cc:2051]: I would leave this as `LOG`

File: src/server/replica.cc [hunk 2]
```diff
@@ -674,7 +682,7 @@ error_code Replica::SendNextPhaseRequest(bool stable) {
   string_view kind = (stable) ? "STARTSTABLE"sv : "SYNC"sv;
   string request = StrCat("DFLY ", kind, " ", master_context_.dfly_session_id);
 
-  LOG(INFO) << "Sending: " << request;
+  DLOG(INFO) << "Sending: " << request;
```
Comment (romange) [inline src/server/replica.cc:685]: similarly `VLOG(1)`

File: src/server/rdb_load.cc [hunk 3]
```diff
@@ -2024,31 +2024,31 @@ error_code RdbLoader::HandleAux() {
...
-    LOG(INFO) << "Loading RDB produced by version " << auxval;
+    DLOG(INFO) << "Loading RDB produced by version " << auxval;
   } else if (auxkey == "ctime") {
     int64_t ctime;
     if (absl::SimpleAtoi(auxval, &ctime)) {
       time_t age = time(NULL) - ctime;
       if (age < 0)
         age = 0;
-      LOG(INFO) << "RDB age " << strings::HumanReadableElapsedTime(age);
+      DLOG(INFO) << "RDB age " << strings::HumanReadableElapsedTime(age);
```
Comment (romange) [inline src/server/rdb_load.cc:2034]: `VLOG(1)` here and below

File: src/server/replica.cc [hunk 4]
```diff
@@ -525,9 +527,11 @@ error_code Replica::InitiateDflySync() {
     return cntx_.ReportError(ec);
   }
 
+  LOG(INFO) << "Started full sync";
+
   // Wait for all flows to receive full sync cut.
   // In case of an error, this is unblocked by the error handler.
-  LOG(INFO) << "Waiting for all full sync cut confirmations";
+  DLOG(INFO) << "Waiting for all full sync cut confirmations";
```
[the remark this answers is not in the document]
Comment (dranikpg) [reply src/server/replica.cc:534]: I'll replace it.
In my mind:
VLOG - we must specifically enable it
DLOG - we had it always enabled, but its restricted to debug builds only. 

Most of the stuff isn't needed as well in debug builds, so we can really vlog it

**Review by romange — APPROVED** [review]

**Pull request merged**